packages/mongodb-chatbot-server/src/test/testConfig.ts

```diff
@@ -141,12 +128,34 @@ ${preprocessedUserMessage ?? originalUserMessage}
   };
 };
 
-export const generateUserPrompt = makeRagGenerateUserPrompt({
-  findContent,
-  queryPreprocessor: mongoDbUserQueryPreprocessor,
-  makeReferenceLinks: makeMongoDbReferences,
-  makeUserMessage,
-});
+export const REJECT_QUERY_CONTENT = "REJECT_QUERY";
+export const NO_VECTOR_CONTENT = "NO_VECTOR_CONTENT";
+export const fakeGenerateUserPrompt: GenerateUserPromptFunc = async (args) => {
```

Comment:
nice 😎

also I love the rhyme of the `make`/`fake` duality

Reply:
![woah](https://media.tenor.com/wS9gJkWOuecAAAAM/coroca-keanu-reeves.gif)

never thought of that